addons/mail/static/src/models/composer_view.js

```diff
@@ -1593,13 +1478,8 @@ registerModel({
         textareaLastInputValue: attr({
```

Comment:
While you are at it, maybe you could reorder this?

Reply:
no useless diff